docs/content/users/topics/sharing.md

```diff
@@ -46,6 +46,27 @@ This set of steps assumes an ngrok subdomain `mg2`:
 * Run `bin/magento setup:store-config:set --base-url="https://mg2.ngrok.io/`.
 * Run [`ddev share`](../usage/commands.md#share) and you’ll see your project at `mg2.ngrok.io`.
 
+## Using localhost.run Service
+
+[https://localhost.run/](https://localhost.run/) is a client-less tool to instantly make a locally running application available on an internet accessible URL.
+
+All major operating systems already have SSH installed, and localhost.run uses SSH as a client, so no download is necessary to use the service and no account setup is needed for free domains.
+
+To connect an internet domain to an application running locally the only thing you’ll need to provide is the **HTTP** port of the DDEV web server.
+
+1. Run `ddev describe` and look for the project’s `http://` URL in the “All URLs” row and identify its port (Refer [`Image`](../../images/ddev-describe.png) for clarity).
```

Comment:
“Refer Image for clarity” isn’t a complete sentence and we typically use backticks to signify variable names, command names, or literal output—I’d avoid all of this by more simply reducing the link to “(Example.)”:

```suggestion
1. Run `ddev describe` and look for the project’s `http://` URL in the “All URLs” row and identify its port ([Example](../../images/ddev-describe.png)).
```